api/gcp/bucket/bucket.go

```diff
@@ -49,22 +86,42 @@ func TransWrite(ctx context.Context, client *storage.Client, path string, reader
 	return
 }
 
-// Delete deletes the given paths in bucket, return the first failed path if any.
-func Delete(ctx context.Context, client *storage.Client, paths ...string) (string, error) {
+// Delete deletes the given paths in bucket.
+func (c *bucketClient) Delete(ctx context.Context, paths ...string) error {
 	ctx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
-	bucketHandler := client.Bucket(config.Config.LDSBucket)
+	bucketHandler := c.client.Bucket(config.Config.LDSBucket)
 	for _, path := range paths {
 		o := bucketHandler.Object(path)
 		if err := o.Delete(ctx); err != nil {
 			if errors.Is(err, storage.ErrObjectNotExist) {
-				log.Printf("Ignore error, file %s does not exist while deleting", path)
+				log.Printf("ignore error, file %s does not exist while deleting", path)
 			} else {
-				log.Printf("Fail to delete file %s in bucket", path)
-				return path, err
+				log.Printf("cloudstore: failed to delete file %s in bucket", path)
```

Comment:
issue: should be "cloudstorage" or just "storage."

Reply:
Fixed typo in the latest commit https://github.com/GoogleCloudPlatform/app-large-data-sharing-golang/pull/34/commits/2c1b6fd19c95db5b7b6714c4611a79f4f617047e